.travis.yml

```diff
@@ -8,11 +8,4 @@ notifications:
         on_failure: always
 
         rooms:
-          digit1819:MI9cZ2mjZzXjEM42jwg4hJph
-
-    slack:
-        on_success: always
-        on_failure: always
-        
-        rooms:
-            digit1819:SG8nfPsbykYZdE3tR07XUUdy
+          digit1819:MI9cZ2mjZzXjEM42jwg4hJph
```

Comment:
Should this be a secret?

Reply:
It can be if we want to, if you want to generate a new one and encrypt it. Didn't feel it really mattered that much.

Reply:
Altough should be noted that this is the same as develop, I only removed the PM to me, getting double messages was starting to get annoying.